wiki/Contests/Mapping_Olympiad/en.md

```diff
@@ -0,0 +1,69 @@
+# osu! Mapping Olympiad
+
+**osu! Mapping Olympiad** is an all-gamemode official contest line-up that was originally from a project of official beatmapping contests revival in 2018. Its particularity is to explore different contests concepts, going from traditional mapping contests to speedmapping or aspire, to mention just a few. 
```

Comment:
```suggestion
**osu! Mapping Olympiad** is an all-gamemode official contest line-up that was originally from a project of official beatmapping contests revival in 2018. Its particularity is to explore different contest concepts, going from traditional mapping contests to speedmapping or aspire, to mention just a few. 
```
would love to see a "History" section detailing what's mentioned here

Reply:
Will look into it with Chaos if he can come up with something about the history of the contest